src/clients/users/index.ts

```diff
@@ -0,0 +1,48 @@
+import { ApiResponse } from "../../api";
+import Knock from "../../knock";
+
+type SetChannelDataInput = {
```

Comment:
One day we really, really need to type this input. cc @connorlindsey

Reply:
Yeah I was thinking about this, I was thinking the right time to do it might be when the slack connector work goes out since that will be introducing the `token` field, and didn't want to do that beforehand.